src/action/index-mobile.js

```diff
@@ -103,141 +99,3 @@ when(
-}));
-store.invoices = [...Array(1)].map(() => ({
-  id: '610da3203c36b17783477cbe5db092220ac7d58477cbe5db092',
-  type: 'lightning',
-  amount: 81345,
-  status: 'in-progress',
-  date: new Date(),
-}));
-store.selectedTransaction = (store.computedTransactions || []).find(
-  tx => tx.type === 'bitcoin'
-);
-store.selectedChannel = store.computedChannels && store.computedChannels[0];
-store.seedMnemonic = [
-  'empower',
-  'neglect',
-  'experience',
-  'elevator',
-  'entropy',
-  'future',
-  'trust',
-  'swift',
-  'pluck',
-  'easy',
-  'kite',
-  'measure',
-  'engage',
-  'settle',
-  'dog',
-  'manager',
-  'tool',
-  'fan',
-  'neglect',
-  'conduct',
-  'blouse',
-  'stone',
-  'quit',
-  'cashew',
-];
-store.logs = [
-  '[14:00:24.995] [info] Using lnd in path lnd',
-  'Checking for update',
-  '[14:00:25.047] [info] lnd: 2018-06-28 14:00:25.039 [WRN] LTND: open /home/valentine/.config/lightning-app/lnd/lnd.conf: no such file or directory',
-  '2018-06-28 14:00:25.039 [INF] LTND: Version 0.4.2-beta commit=884c51dfdc85284ba8d063c4547d2b5665eba010',
-  '2018-06-28 14:00:25.039 [INF] LTND: Active chain: Bitcoin (network=testnet)',
-  '2018-06-28 14:00:25.039 [INF] CHDB: Checking for schema update: latest_version=1, db_version=1',
-  '[14:00:25.170] [info] lnd: 2018-06-28 14:00:25.055 [INF] RPCS: password RPC server listening on 127.0.0.1:10009',
-  '2018-06-28 14:00:25.055 [INF] RPCS: password gRPC proxy started at 127.0.0.1:8080',
-  '2018-06-28 14:00:25.055 [INF] LTND: Waiting for wallet encryption password. Use `lncli create` to create a wallet, `lncli unlock` to unlock an existing wallet, or `lncli changepassword` to change the password of an existing wallet and unlock it.',
-  '[14:00:25.541] [info] Loaded initial state',
-  '[14:00:25.557] [info] GRPC unlockerReady',
-  'Found version 0.2.0-prealpha.9 (url: Lightning-linux-x86_64v0.2.0-prealpha.9.AppImage)',
-  'Downloading update from Lightning-linux-x86_64v0.2.0-prealpha.9.AppImage',
-  'No cached update available',
-  'File has 2893 changed blocks',
-  'Full: 106,265.24 KB, To download: 59,575.39 KB (56%)',
-  'Differential download: https://github.com/lightninglabs/lightning-app/releases/download/v0.2.0-prealpha.9/Lightning-linux-x86_64v0.2.0-prealpha.9.AppImage',
-  'Redirect to https://github-production-release-asset-2e65be.s3.amazonaws.com/76898197/428914b4-7561-11e8-8826-08fde1bd29aa',
-  '[14:00:33.730] [info] lnd: 2018-06-28 14:00:33.730 [INF] LNWL: Opened wallet',
-  '[14:00:33.731] [info] lnd: 2018-06-28 14:00:33.730 [INF] LTND: Primary chain is set to: bitcoin',
-  '[14:00:33.879] [info] lnd: 2018-06-28 14:00:33.879 [INF] BTCN: Loaded 1032 addresses from file /home/valentine/.config/lightning-app/lnd/data/chain/bitcoin/testnet/peers.json',
-  '[14:00:33.893] [info] lnd: 2018-06-28 14:00:33.892 [INF] CMGR: DNS discovery failed on seed x49.seed.tbtc.petertodd.org: lookup x49.seed.tbtc.petertodd.org: No address associated with hostname',
-].join('\n');
```

Comment:
@halseth can you rebase on the current `eject-to-expokit` branch? This diff shouldn't be in here anymore.